crates/examples/tests/testfiles.rs

```diff
@@ -12,10 +30,11 @@ fn testfiles() {
     let mut fail = false;
     for dir in glob::glob("testfiles/*").unwrap().filter_map(Result::ok) {
         let dir = dir.to_str().unwrap();
-        for path in glob::glob(&format!("{}/*", dir))
+        let matches = glob::glob(&format!("{}/*", dir))
             .unwrap()
-            .filter_map(Result::ok)
-        {
+            .filter_map(Result::ok);
+
+        for path in skip_wasm_test_if_unsupport(matches) {
```

Comment:
It'd be better if `skip_wasm_test_if_unsupport` was a filter that returned `bool`, so that we could do:
```rust
   for path in glob::glob(&format!("{}/*", dir))
        .unwrap()
        .filter_map(Result::ok)
        .filter(skip_wasm_test_if_unsupported)
```

Reply:
Yes it's better, but I somehow didn't come up with this idea.

Code updated.